docs/sdks/rust/cache.mdx

```diff
@@ -0,0 +1,97 @@
+---
+sidebar_position: 1
+sidebar_label: Cache
+title: Getting started with Momento Cache in Rust
+description: Learn the basic building blocks for writing Rust code to interact with Momento Cache.
+keywords:
+  - momento
+  - cache
+  - caching
+  - topics
+  - pubsub
+  - pub/sub
+  - eda
+  - event-driven architecture
+  - redis
+  - serverless
+  - sdk
+  - sdks
+  - api
+  - examples
+  - resources
+  - getting started
+  - quick start
+  - rust
+  - tokio
+---
+
+import { SdkExampleCodeBlock } from "@site/src/components/SdkExampleCodeBlock";
+// This import is necessary even though it looks like it's un-used; The inject-example-code-snippet
+// plugin will transform instances of SdkExampleCodeBlock to SdkExampleCodeBlockImpl
+import { SdkExampleCodeBlockImpl } from "@site/src/components/SdkExampleCodeBlockImpl";
+
+# Getting Started with Momento Cache in Rust
+
+If you need to get going quickly with Rust and Momento Cache, this page contains the basic API calls you'll need. [Check the Rust SDK examples](https://github.com/momentohq/client-sdk-rust/tree/main/example) for complete, working examples including build configuration files.
+
+## Install the Momento SDK
+
+The Momento SDK is available on crates.io: [`momento`](
+https://crates.io/crates/momento).
+
+:::tip
+Visit [crates.io](https://crates.io/crates/momento) to find the latest available version of the SDK.
+:::
+
+Install the client library in an existing Rust project:
+
+```bash
+cargo add momento
+```
+
+## Set up your API key
+
+You'll need a Momento API key to authenticate with Momento.  You can get one from the [Momento Web Console](https://console.gomomento.com/caches).
+Once you have your API key, store it in an environment variable so that the Momento client can consume it:
+
+```
+export MOMENTO_API_KEY=<your Momento API key here>
+```
+
+## Import libraries and instantiate a CacheClient object
+
+This code sets up the main function, the necessary imports, and the CacheClient instantiation that each of the other functions will need to call.
+
+<SdkExampleCodeBlock language={'rust'} file={'./example/src/bin/cheat_sheet.rs'} />
```

Comment:
missing cheat sheet? or should this be `cache.rs`?

Reply:
see this PR: https://github.com/momentohq/client-sdk-rust/pull/317

Reply:
oops i might have somehow missed adding the file to that PR, double-checking.